### ISSUE TITLE
Sample State

### ISSUE THREAD

**Issue by andrewlangston** [nkossally/Poll_Anywhere#4]
Najja,

Good preliminary work on your sample state. I will have further comments once it is fleshed out further, so for now please see one note below:

- [x] It looks like many pieces of data are missing, such as responses, answer choices, etc. Please finalize the sample state and I will review further. Thanks!

**Comment by nkossally** [comment]
I made some edits. Let me know what you think.

**Comment by andrewlangston** [comment]
Najja - this is good progress! One comment for now:
- [ ] I don't think you need to store a response's `screen_name` in state. Instead, recall that the `responses` table is a joins table for `users` and `choices`. So, `responses` should just hold a user id and a choice id. We'll be able to get to `polls` through that choice id. Let me know if you have questions about this.

**Comment by nkossally** [comment]
Yeah. I thought the screen_name would be some chosen screen name of anyone responding to a poll, not the name or username of the creator o the poll.

**Comment by andrewlangston** [comment]
Najja - thanks for the response. Let's keep things straightforward for now and remove `screen_name`, especially if it is not information we are persisting to the database. We can simply rely on a user's `username` for this purpose.
